include/swift/SIL/SILWitnessTable.h

```diff
@@ -89,7 +103,9 @@ class SILWitnessTable : public llvm::ilist_node<SILWitnessTable>,
     Method,
     AssociatedType,
     AssociatedTypeProtocol,
-    BaseProtocol
+    BaseProtocol,
+    // SWIFT_ENABLE_TENSORFLOW
```

Comment:
I can understand why you want these comments in your fork, maybe, but please strip them when upstreaming.

Reply:
Wait, no, this patch is just going into your fork.  Why do you want these comments?  It seems like nothing but ugly bookkeeping.

Reply:
It's a tradition/requirement started by Chris when he made the fork. I expect to merge AD stuff upstream ASAP because it's becoming increasingly invasive, possibly starting at the EOY.